.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - run:
           name: install dependencies
           environment:
-            OTC_BUILDER_VERSION: 0.73.0
+            OTC_BUILDER_VERSION: 0.95.0
```

Comment:
any reason not to bump this to 0.98.0?

Reply:
Right i guess my question was why not take the 0.98.0 dependencies

Reply:
Bumped